week3/homework/src/index.js

```diff
@@ -1,3 +1,99 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const fs = require('fs');
+const express = require('express');
+const bodyParser = require('body-parser');
+const functions = require('./functions.js');
+const app = express();
+app.use(bodyParser.json());
+
+app.get('/todos/:id', (request, response) => {
+  const index = request.params.id;
+  fs.readFile('./src/to-do.json', 'utf8', (error, data) => {
+    const todos = data ? JSON.parse(data) : [];
+    if (error) {
+      console.log(error);
+      response
+        .status(500)
+        .json({ result: 'error' });
+      return;
+    }
+    response
+      .status(200)
+      .json(functions.showTask(todos, index));
+  });
+});
+app.delete('/todos', (request, response) => {
+  functions.reset();
+  response
+    .status(301)
```

Comment:
`301` is the status code for moved permanently. Look at https://en.wikipedia.org/wiki/List_of_HTTP_status_codes